feature/home/src/main/java/com/epmedu/animeal/home/data/model/FeedingPoint.kt

```diff
@@ -0,0 +1,17 @@
+package com.epmedu.animeal.home.data.model
+
+import com.epmedu.animeal.common.data.model.MapLocation
+import com.epmedu.animeal.foundation.switch.AnimalType
+import com.epmedu.animeal.home.data.model.enum.AnimalPriority
+import com.epmedu.animeal.home.data.model.enum.AnimalState
+import com.epmedu.animeal.home.data.model.enum.Remoteness
+
+data class FeedingPoint(
+    val id: Int, // For future implementations
+    val animalPriority: AnimalPriority,
```

Comment:
What is the purpose of priority? I don't see the usage of it in the code

Reply:
The same as remoteness, nothing for now. If you see https://jira.epam.com/jira/browse/EPMEDU-731 in the requirements list it is specified.